agent/provider/src/cli/keystore.rs

```diff
@@ -0,0 +1,91 @@
+use std::collections::HashSet;
+use std::path::{PathBuf};
+
+use structopt::StructOpt;
+
+use ya_manifest_utils::{util, Keystore, KeystoreLoadResult};
+
+use crate::startup_config::ProviderConfig;
+
+#[derive(StructOpt, Clone, Debug)]
+#[structopt(rename_all = "kebab-case")]
+pub enum KeystoreConfig {
+    /// List trusted X.509 certificates
+    List,
```

Comment:
Probably should add `Show` to display equivalent of `openssl x509 -text`, because `List` does not show all the data.